test/distributed/fsdp/test_fsdp_core.py

```diff
@@ -27,7 +27,7 @@
     run_tests,
 )
 
-from torch.distributed.fsdp import CPUOffload
+from torch.distributed.fsdp import CPUOffload, MixedPrecision
```

Comment:
fairscale version used shardedGradScaler and autocast for DDP tests, and then compare fsdp results with DDP. could we add follow up tests for these?

Reply:
Yes, I think the first step would be to build/import the shardedgradscaler from fairscale.

Reply:
We should, although we haven't built ShardedGradScaler at the moment. Will add this in follow up changes.

Reply:
> used shardedGradScaler and autocast for DDP tests

ShardedGradScaler would not apply to DDP, is that correct?